faunadb/query.py

```diff
@@ -234,6 +234,26 @@ def delete(ref):
   """See the `docs <https://faunadb.com/documentation/queries#write_functions>`__."""
   return {"delete": ref}
 
+
+def insert(ref, ts, action, params):
```

Comment:
I think we should normalize the order of the args here with the Event constructor (probably by updating the Event constructor to match this?)